Art/KC1064-Welcome/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome-Animation</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
## Stylesheet name
- [ ] your css file should be named `styles.css` 
- [ ] make sure you adjust this ling tag accordingly after changing the name

Reply:
Sure....I will resolve the issue and raise a PR again. Thanks for the review and feedback.